.circleci/src/@common.yml

```diff
@@ -8,7 +8,19 @@ parameters:
   run_commit:
     default: true
     type: boolean
-  run_backend:
+  run_commit_main:
```

Comment:
@ztefanie do i have to register this somewhere to be executed on commit on main?
I think you've showed sth in your presentation but couldn't find it

Reply:
No it is enough to specify the condition in the beginning of the workflow.